components/org.wso2.identity.webhook.common.event.handler/src/main/java/org/wso2/identity/webhook/common/event/handler/internal/handler/CredentialEventHookHandler.java

```diff
@@ -196,4 +207,33 @@ public boolean isCredentialUpdateFlow(String eventName) {
 
         return IdentityEventConstants.Event.POST_UPDATE_CREDENTIAL_BY_SCIM.equals(eventName);
     }
+
+    private void publishCredentialEvent(String tenantDomain, Channel credentialChangeChannel, String eventUri,
```

Comment:
I think we can name this as publishCredentialEventForCurrentOrg or something similar